.github/workflows/e2e-tests-preview.yml

```
@@ -0,0 +1,686 @@
+      BREVO_WEBHOOK_SECRET: ${{ secrets.BREVO_WEBHOOK_SECRET }}
+      # Vercel Configuration (for deployment operations)
+      VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+      VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+      VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+      # GitHub Token (automatically provided by GitHub Actions)
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Test Environment Configuration
+      E2E_TEST_MODE: 'true'
+      NODE_ENV: 'test'
+      # Dynamic URL variables (set by target-url step)
+      PLAYWRIGHT_BASE_URL: ${{ env.PLAYWRIGHT_BASE_URL }}
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)